docs/source/releasenotes/0.14.0.rst

```diff
@@ -0,0 +1,283 @@
+====================
+What's New in 0.14.0
+====================
+
+.. currentmodule:: openmc
+
+-------
+Summary
+-------
+
+This release of OpenMC includes many bug fixes, performance improvements, and
+several notable new features. Some of the highlights include projection plots,
+pulse height tallies for photons, weight window generation, and an ability to
+specify continuous removal or feed of nuclides/elements during depletion.
+Additionally, one of the longstanding annoyances of depletion calculations,
+namely the need to include initial "dilute" nuclides, has been eliminated. There
+are also a wide array of general improvements in the Python API.
+
+------------------------------------
+Compatibility Notes and Deprecations
+------------------------------------
+
+- The :class:`openmc.deplete.MicroXS` has been completely redesigned and
+  improved. See further comments below under "New Features". (`#2572
+  <https://github.com/openmc-dev/openmc/pull/2572>`_, `#2579
+  <https://github.com/openmc-dev/openmc/pull/2579>`_, `#2595
+  <https://github.com/openmc-dev/openmc/pull/2595>`_, `#2700
+  <https://github.com/openmc-dev/openmc/pull/2700>`_)
+- The ``rectangular_prism`` function has been replaced by the
+  :class:`openmc.model.RectangularPrism` class and the ``hexagonal_prism``
+  function has been replaced by the :class:`openmc.model.HexagonalPrism` class.
+  Note that whereas the ``rectangular_prism`` and ``hexagonal_prism`` functions
+  returned a region representing the interior of the prism, the new
+  :class:`~openmc.model.RectangularPrism` and
+  :class:`~openmc.model.HexagonalPrism` classes return composite surfaces, so
+  you need to use the unary ``-`` or ``+`` operators to obtain a region that can
+  be assigned to a cell. (`#2739
+  <https://github.com/openmc-dev/openmc/pull/2739>`_)
+- The ``Source`` class has been refactored and split up into three separate
+  classes: :class:`~openmc.IndependentSource`, :class:`~openmc.FileSource`, and
+  :class:`~openmc.CompiledSource`. (`#2524
+  <https://github.com/openmc-dev/openmc/pull/2524>`_)
+- The ``vertices`` and ``centroids`` attributes on mesh classes now always
+  return Cartesian coordinates. (`#2711
```

Comment:
```suggestion
- The ``vertices`` and ``centroids`` attributes on mesh classes now always
  return Cartesian coordinates and the shape of the returned arrays has changed to allow `ijk` indexing using a tuple (i.e. `xyz = vertices[i, j, k]`). (`#2711
```

Reply:
Thanks @pshriwise -- just incorporated this suggestion.